custom/icds_reports/tests/outputs/ccs_record_monthly_2017-04-01_sorted.csv

```diff
@@ -21,7 +21,7 @@ a1,b69e4bd0-5362-4441-9389-855422bb9dd1,2017-04-01,268,other,0,1,,0,0,0,0,0,0,0,
 a1,b9696ca3-8ab5-46fe-a1be-7d25d9692e33,2017-04-01,364,other,1,1,,0,0,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,2016-05-01,0,obc,no,no,yes,,,,,,,,,,,,,,,,2017-08-08,,,,,,,,,,0,,,,,,,2017-04-12,,1987-01-12,0,390c54b8-e463-43c1-ab8a-8646be958f5f
 a1,c498b719-cad3-4be2-b342-a9bd35128714,2017-04-01,304,other,1,1,,0,0,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,2016-05-01,0,obc,no,no,yes,,,,,,,,,,,,,,,,2017-08-08,,,,,,,,,,0,,,,,,,2017-04-12,,1992-01-10,0,9e8de64b-b6e6-482e-a71d-8d266d55e7bd
 a1,cc2e1551-94d4-41dc-a7be-cb1c78c4f42b,2017-04-01,268,other,1,1,,0,0,0,0,0,0,0,0,,0,0,0,0,0,1,0,0,0,,0,0,0,0,0,0,0,0,0,0,0,0,0,0,,0,0,0,0,0,2016-05-01,0,obc,no,no,yes,,,,,,,,,,,,,,,,2017-08-08,,,,,,,,,,0,,,,,,,2017-04-12,,1995-01-08,0,a361753c-9ee5-408b-ae6b-ab807424a81d
-a1,d499215f-5e4e-4c97-8ea5-78c3a9984881,2017-04-01,304,lactating,1,1,,20,1,0,0,0,0,0,0,,0,0,1,0,0,0,1,1,0,,0,0,0,0,0,0,0,0,0,0,1,0,0,1,,0,0,1,1,0,2017-03-01,0,obc,no,no,yes,,,,,,,,,,,1,1,1,,,2017-08-08,,,,,,,,,,1,,,,,,,2017-04-12,,1992-01-10,0,dde249e4-a7b1-4d2b-bee3-d0e93a8566bf
+a1,d499215f-5e4e-4c97-8ea5-78c3a9984881,2017-04-01,304,lactating,1,1,,20,1,0,0,0,0,0,0,,0,0,1,0,0,0,0,0,0,,0,0,0,0,0,0,0,0,0,0,1,0,0,1,,0,0,1,1,0,2017-03-01,0,obc,no,no,yes,,,,,,,,,,,1,1,1,,,2017-08-08,,,,,,,,,,1,,,,,,,2017-04-12,,1992-01-10,0,dde249e4-a7b1-4d2b-bee3-d0e93a8566bf
```

Comment:
could have changed the pregnant instead of resting_in_pregnancy and extra_meal But pregnant parameter is used in other places in icds also like pregnant and lacting women report. Did not want to propagate the change